README.md

```diff
@@ -53,23 +53,29 @@ Users with administrative credentials can configure the plugin through the stand
 
 Once setup and configured, the Decentralized Storage is ready to use.
 
-- Access the Decentralized Storage folder under "All Files" or via the "Files" menu under "External Storage."
-- Upload files as you would in Nextcloud.
+-   Access the Decentralized Storage folder under "All Files" or via the "Files" menu under "External Storage."
+-   Upload files as you would in Nextcloud.
 
 ![Upload file to swarm in Nextcloud](https://github.com/user-attachments/assets/aadd664e-26ca-470a-a27b-af8d94351e52)
 
-- Once uploaded, it can be viewed internally or downloaded. It can also be copied to another storage.
+-   Once uploaded, it can be viewed internally or downloaded. It can also be copied to another storage.
 
 ![View, download, and file view actions in Nextcloud](https://github.com/user-attachments/assets/596b72ed-d97f-48ba-bcb9-0ee5ff581a3c)
 
-- Files from another Nextcloud storage can also be copied/moved to the decentralized storage.. Choose the decentralized storage folder as the target Folder
+-   Files from another Nextcloud storage can also be copied/moved to the decentralized storage.. Choose the decentralized storage folder as the target Folder
 
 ![Copying or moving files to decentralized storage folder](https://github.com/user-attachments/assets/ceed3585-f7e6-4f16-b371-d61402e9f1e9)
 
-- Right-click on a Swarm file to copy the Swarm reference (hash) to your clipboard. Alternatively, click the three dots in the Actions menu and select 'Copy Swarm Reference.' The Swarm reference is the unique address of the file on the Swarm network.
+-   Right-click on a Swarm file to copy the Swarm reference (hash) to your clipboard. Alternatively, click the three dots in the Actions menu and select 'Copy Swarm Reference.' The Swarm reference is the unique address of the file on the Swarm network.
 
 ![Copy swarm reference to clipboard](https://github.com/user-attachments/assets/cc73282b-e32e-411f-a94b-a2ac3313f60b)
 
+### Additional Integration with Moodle
+
+You can also enhance your experience by integrating the Hejbit Swarm plugin with the **Moodle Nextcloud plugin**. This integration allows users to access decentralized storage directly within the Moodle environment, providing a seamless experience for educational content management. For more information on how to set up this integration, visit the ![Moodle Nextcloud repository documentation](https://docs.moodle.org/405/en/Nextcloud_repository).
+
+For detailed guidance on using the integrated features, please refer to the ![Metaprovide Learning Platform](https://learning.metaprovide.org/login/index.php).
+
```

Comment:
The changes to the README can be seen immediately on github in the branch https://github.com/MetaProvide/nextcloud-swarm-plugin/tree/FIX/%231085--adding-moodle-to-the-documentation - 

The 2 external links do not open. The links show a "Non-Image content-type returned" message.
This is because the ! character identifies the resource as an image. If you remove the ! before the [], it should fix it, and the markup will treat it as a link.